app/javascript/components/shared_components/BackgroundBuffer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import { useLocation } from 'react-router-dom';
+
+export default function BackgroundBuffer() {
+  const location = useLocation();
+
+  if (location?.pathname.startsWith('/rooms/')) {
```

Comment:
What happens if you go to `localhost:3000/rooms/`? I think we need to make this a bit smarter

Reply:
Nice catch, I'll see what we could do.